examples/ticket-management-system/main.bal

```diff
@@ -0,0 +1,145 @@
+import ballerina/io;
+import ballerina/oauth2;
+import ballerina/http;
+import ballerinax/hubspot.crm.obj.tickets as hstickets;
+
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string refreshToken = ?;
+
+string createdTicketId = "";
+string batchTicketId1 = "";
+string batchTicketId2 = "";
+string batchTicketId3 = "";
+
+hstickets:OAuth2RefreshTokenGrantConfig auth = {
+    clientId,
+    clientSecret,
+    refreshToken,
+    credentialBearer: oauth2:POST_BODY_BEARER 
+};
+
+final hstickets:Client TicketClient = check new hstickets:Client({auth});
```

Comment:
Update the variable accordingly. Treat the `client` as another variable, it shouldn't start with a cap.

```suggestion
final hstickets:Client hubspot = check new ({auth});
```